test/functional/features/cma-requirements-listed.feature

```diff
@@ -0,0 +1,11 @@
+@cmaRequirements
+Feature: Cma Requirements Listed @only
```

Comment:
Is this only tag intended?

Reply:
Ah yes that just added to run the that test locally, I shall remove